website_and_docs/content/documentation/webdriver/browsers/chrome.zh-cn.md

```diff
@@ -64,7 +64,9 @@ Add a browser location to options:
 
 ### Add extensions
 
-The `extensions` parameter accepts crx files
+The `extensions` parameter accepts crx files. As for unpacked directories,
+please use the `load-extension` argument instead, as mentioned in
+[this post](https://chromedriver.chromium.org/extensions).
```

Comment:
I did not know about this... This might explain why some of the Selenium repo tests fail with the unpacked directories...

Reply:
Yes, `addExtensions` doesn't seem to work for unpacked extensions (directories).